RSC Publishing.js

```diff
@@ -73,18 +73,21 @@ function scrape(doc, type) {
 		if(item.tags.length == 1 && !item.tags[0]) {
 			item.tags = [];
 		}
-		
+
 		if (item.date) {
 			item.date = ZU.strToISO(item.date);
 		}
 
-		if (item.itemType == "bookSection") {
-			if (item.DOI){
-				if (item.extra) item.extra += "\nDOI: " + item.DOI;
-				else item.extra = "DOI: " + item.DOI;
+		if (item.DOI && !ZU.fieldIsValidForType("DOI", item.itemType)){
+			if (item.extra) {
+				if (item.extra.search(/^DOI:/) == -1) {
+					item.extra += "\nDOI: " + DOI;
+				}
+			} else {
+				item.extra = "DOI: " + DOI;
```

Comment:
We could also change the `itemType` in the translatorObject cf. https://github.com/zotero/translators/blob/master/Embedded%20Metadata.js#L815 and then (I assume) it is not necessary anymore. I.e.
```js

	translator.getTranslatorObject(function(trans) {
		trans.itemType = type;
		trans.doWeb(doc, url);
	});
```